lib/Doctrine/ORM/Tools/Console/Command/ValidateSchemaCommand.php

```diff
@@ -76,7 +76,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         if ($input->getOption('skip-mapping')) {
             $output->writeln('<comment>[Mapping]  Skipped mapping check.</comment>');
-        } elseif ($errors = $validator->validateMapping()) {
+        } elseif (($errors = $validator->validateMapping()) === false) {
+            $output->writeln('<error>[Mapping]  FAIL - No entities defined!</error>');
+        } elseif (count($errors) > 0) {
```

Comment:
Now by looking at this I think $error won't be defined here. I should really write a test for the `ValidateSchemaCommand`.